proto/beacon/p2p/v1/types.proto

```diff
@@ -167,16 +167,17 @@ message Crosslink {
 }
 
 message BeaconBlock {
+  // Deprecated
+  // TODO(2307): Remove deprecated fields
+  bytes randao_reveal = 1003;
+  Eth1Data eth1_data = 1004;
+  // 0.6 fields
```

Comment:
remove 0.6 fields

Reply:
replaced with // End of deprecated fields

Reply:
reverted